app/mailers/producer_mailer.rb

```diff
@@ -12,18 +12,28 @@ def order_cycle_report(producer, order_cycle)
 
     subject = "[#{Spree::Config.site_name}] Order cycle report for #{producer.name}"
 
+    # Set from address to the coordinator (optional)
+    if Spree::MailMethod.current
+      if Spree::MailMethod.current.prefers_send_from_enterprise_address?
+        from_email = @coordinator.email
+      else
+        from_email = from_address
+      end
+    end
+    from_email ||= from_address
+
     if has_orders? order_cycle, producer
       mail(to: @producer.email,
-           from: from_address,
+           from: from_email,
            subject: subject,
-           reply_to: @coordinator.email,
```

Comment:
@stveep, I don't think we should be removing this, as the reply-to address should be the shop email address, regardless of which address the email is sent from... right?

For example, I regularly get people replying to their order confirmation emails (and I guess producer emails are in much the same boat). If those replies start going to `Spree::Config.emails_sent_from`, I am never going to see them...